app/src/main/kotlin/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModel.kt

```diff
@@ -1850,6 +1972,39 @@ class VaultItemListingViewModel @Inject constructor(
         }
     }
 
+    private fun verifyUserAndAuthenticatePasswordCredential(
+        request: ProviderGetCredentialRequest,
+        selectedCipher: CipherView,
+    ) {
+
+        if (bitwardenCredentialManager.isUserVerified) {
+            authenticatePasswordCredential(
+                request = request,
+                cipherView = selectedCipher,
+            )
+            return
+        }
+
+        val userVerificationRequirement =
+            bitwardenCredentialManager.getUserVerificationRequirement(request)
```

Comment:
The `UserVerificationRequirement` is exclusive to the FIDO 2 spec, so we don't need to check it when processing GetPasswordCredential requests. The only thing that should interrupt the user at this point it the master password reprompt if it is required for the selected cipher.

Reply:
simlified it (removed the authenticate function and basically just send the event here)